layouts/page/about.html

```diff
@@ -132,7 +132,7 @@ <h2 class="text-center mb-8">Awards &amp; Recognitions</h2>
             <ul class="list-none block p-0 inline-flex flex-wrap justify-center">
                 {{ range first 6 (where $.Site.Data.awards.awards "featured" true) }}
                     <li class="mx-4 bg-white">
-                        <a href="{{ .url }}">
+                        <a data-track="award-{{ .url | urlize }}" href="{{ .url }}">
```

Comment:
Same here -- `award` is probably what you want, or suchlike.

Reply:
See above. The intention is that changes in the content will change the ids so the data is a bit easier for us to consume (more human-readable).